lib/developer_portal/lib/liquid/filters/rails_helpers.rb

```diff
@@ -50,9 +51,9 @@ def javascript_include_tag(name, options = {})
         js = RailsHelpers.replace_googleapis(name)
         case
         when THREESCALE_WEBPACK_PACKS.include?(name) # TODO: This is an intermediate step in order to tackle webpack assets in dev portal. A final solution might be needed easing the update of templates/assets.
-          view.javascript_pack_tag(name, options)
+          active_docs_proxy(name) + view.javascript_pack_tag(name, options)
```

Comment:
I'm not entirely sure if the webpack version (which I assume it uses the latest swaggerUI) needs it, but, I also imagine you tried it and it was in the end.

Reply:
I leave it there in case :)

Reply:
no harm I guess hehe